src/MapPanel/styled.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import MaterialCard from '@material-ui/core/Card'
+import MaterialTabs from '@material-ui/core/Tabs'
+import MaterialTab from '@material-ui/core/Tab'
+import MaterialCheckbox from '@material-ui/core/Checkbox'
+import MaterialCardContent from '@material-ui/core/CardContent'
+import { styled } from '@material-ui/styles'
+
+export const Card = styled(({ ...props }) => <MaterialCard {...props} />)({
+  position: props => props.inline ? 'inline' : 'absolute',
```

Comment:
inline is not a thing for this card...always absolute b/c we're positioned absolute to the map container